### ISSUE TITLE
[From Market] java.lang.NullPointerException in XmppManager.onConnectionComplete()

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#104]
```
java.lang.NullPointerException
at 
com.googlecode.gtalksms.XmppManager.onConnectionComplete(XmppManager.java:374)
at com.googlecode.gtalksms.XmppManager.initConnection(XmppManager.java:332)
at com.googlecode.gtalksms.XmppManager.start(XmppManager.java:126)
at com.googlecode.gtalksms.XmppManager.start(XmppManager.java:120)
at 
com.googlecode.gtalksms.MainService.xmppRequestStateChange(MainService.java:457)
at com.googlecode.gtalksms.MainService.onHandleIntent(MainService.java:191)
at 
com.googlecode.gtalksms.MainService$ServiceHandler.handleMessage(MainService.jav
a:103)
at android.os.Handler.dispatchMessage(Handler.java:99)
at android.os.Looper.loop(Looper.java:123)
at android.os.HandlerThread.run(HandlerThread.java:60)

GTalkSMS version =1.9
```

Original issue reported on code.google.com by `Florent....@gmail.com` on 13 Feb 2011 at 11:50

**Comment by GoogleCodeExporter** [comment]
```
Crashes due to xmpp roster exceptions
```

Original comment by `Florent....@gmail.com` on 13 Feb 2011 at 12:01
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v1.9.1
```

Original comment by `Florent....@gmail.com` on 13 Feb 2011 at 12:48
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:50
- Changed state: **Deployed**
